src/util/StreamLink.cpp

```diff
@@ -272,6 +272,16 @@ void openStreamlinkForChannel(const QString &channel)
         args << "--stream-sorting-excludes" << exclude;
     }
 
+    // Append any extra options to our streamlink command
+    if (getSettings()->streamlinkOptsLatency)
+    {
+        args << "--twitch-low-latency";
+    }
+    if (getSettings()->streamlinkOptsAds)
+    {
+        args << "--twitch-disable-ads";
+    }
+
```

Comment:
We should not be suggesting options for use with streamlink

Reply:
Good point, I meant to default them to false / disabled. Should be fixed now.

Reply:
> suggesting options

This still suggests options, he means remove the suggestions.

Reply:
Oh, then I would disagree. There are only two options streamlink provides besides disabling of reruns. The user should be able to quickly toggle them if they want to. Each time I do a fresh install I need to find these arguments. I don't see why people would not want this.

Reply:
> Oh, then I would disagree. There are only two options streamlink provides besides disabling of reruns. The user should be able to quickly toggle them if they want to. Each time I do a fresh install I need to find these arguments. I don't see why people would not want this.

Because one of your suggestions is very obviously something Twitch does not want to support, and we aren't going to bring attention to said frowned upon functions from another application, in this one. 
If we were going to spit in Twitch's face I'd much rather do it in better functional ways.